### DISCUSSION TITLE
[Study suggestion] AWS study / practice

### DISCUSSION THREAD

**Discussion by swiri021** [OpenKBC/multiple_sclerosis_proj#63] [Study]
Thinking about new study topic for the project, engineering part: AWS batch (AWS)

**Comment by swiri021** [comment]
* Including CloudFormation

**Reply by lacuss** [reply]
If it is helpful, I can ask about the elastic service of AWS intro with Mike(aka. my husband) about maintaining low cost in AWS. He has been using AWS for more than 10 yrs. info about Mike - https://docs.google.com/document/d/1G9FieEGlqfZXUUKAkTpI7LFG621moyaJB2IwXiR7rX8/edit

**Reply by swiri021** [reply]
That would be awesome!! Nicole, if he has a time, we can set up special seminar time for him.
